pkg/bosh/manifest/manifest_test.go

```diff
@@ -0,0 +1,409 @@
+package manifest_test
+
+import (
+	. "code.cloudfoundry.org/cf-operator/pkg/bosh/manifest"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	yaml "gopkg.in/yaml.v2"
+)
+
+var _ = Describe("Manifest", func() {
+	var (
+		job                 Job
+		jobY                []byte
+		instanceGroup       InstanceGroup
+		instanceGroupY      []byte
+		manifest            Manifest
+		manifestY           []byte
```

Comment:
Do these have to be global, they're not shared between contexts and not used in `BeforeEach`.

Also  the 'Y' suffix confused me while reading.